src/components/Douga.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <normal :welcomeTexts="welcomeTexts" v-if="screenMode === 'normal'"></normal>
+    <tablet :welcomeTexts="welcomeTexts" v-else-if="screenMode === 'tablet'"></tablet>
+    <mobile :welcomeTexts="welcomeTexts" v-else></mobile>
```

Comment:
@phann-phearoen 

Please refer to the feedbacks on your PR(#2) .
[Vuetify Grid System] is recommended for both resizing and device responsiveness simultaneously.

Reply:
There's a problem with that because Vutify is not compatible with Vue3 yet! It said to be released later this year.